### PR TITLE
update flake.nix to 21.11

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

**Review by regnat — APPROVED** [review]

File: flake.nix [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   description = "The purely functional package manager";
 
-  inputs.nixpkgs.url = "nixpkgs/nixos-21.05-small";
+  inputs.nixpkgs.url = "nixpkgs/nixos-21.11-small";
   inputs.lowdown-src = { url = "github:kristapsdz/lowdown"; flake = false; };
```
Comment (Mic92) [inline flake.nix:5]: Not sure if this is still required?

**Comment by edolstra** [comment]
Unfortunately I had to revert this for now because util-linux is broken in the musl build (https://hydra.nixos.org/build/161448992). So we should fix that in nixpkgs first.

**Comment by alyssais** [comment]
We already did, but the proper fix is https://github.com/NixOS/nixpkgs/commit/2ebeb02a99aab4fc6813fe401e2d375a5c00bc8a, which is a mass rebuild, so I'm not sure about backporting it.  Thoughts?

**Comment by Mic92** [comment]
Mhm. Would targeting unstable for the musl build acceptable?

**Pull request merged**